### ISSUE TITLE
Narrator: The narrator does not announces the link present in Callout content.

### ISSUE THREAD

**Issue by rajnish07** [microsoft/fluentui#14942]
#url: https://codepen.io/rajnish07/pen/ZEWrqLa?editors=0010

### Environment Information

Windows Version - Win10 
Chromium Edge Version: 85.0.552.1 (Official Build) dev (64-bit)

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/rajnish07/pen/ZEWrqLa?editors=0010

#### Repro Steps

          1. Start the narrator.
          2. Navigate to Show Callout using tab key and press enter.
          3. Observe that the narrator is not announcing the “vitae dicta sunt explicabo” link while navigating using the tab key.

#### Actual behavior:

While navigating using the tab key, the narrator is not announcing the “vitae dicta sunt explicabo” link, which is present in the Show Callout content.
![image](https://user-images.githubusercontent.com/40293642/92544802-5f734100-f26c-11ea-8923-5c07882f3124.png)

#### Expected behavior:
The narrator should announce the “vitae dicta sunt explicabo” link while navigating using tab key.

### Documentation describing expected behavior

**Note: This issue is not observed with Chrome+NVDA

**Comment by paulgildea** [comment]
@rajnish07 Thanks for the feedback.

@smhigley FYI

**Comment by kolaps33** [comment]
Looking to the screenshot and I see focus goes on the whole "popup". 
I am thinking if is necessary to focus whole "popup" ? (because generally focus should go only to actionable elements not to text elements)
In my opinion it would be enough if focus land only on the link inside the "popup".  User can then use virtual cursor/scan mode to go through the text inside the "popup".
And if the focus goes directly to the link, it could possibly fix as well narration of the link, because reader will be not polluted with narrating the whole popup. But not saying that this is the main reason why link is not narrating.
